sunbeam-python/sunbeam/plugins/validation/README.md

```diff
@@ -0,0 +1,31 @@
+# Validation
+
+This plugin provides OpenStack Integration Test Suite: tempest for Sunbeam. It is based on [tempest-k8s](https://opendev.org/openstack/sunbeam-charms/src/branch/main/charms/tempest-k8s) and [tempest-rock](https://github.com/canonical/ubuntu-openstack-rocks/tree/main/rocks/tempest) project.
+
+## Installation
+
+To enable cloud validation, you need an already bootstrapped Sunbeam instance. Then, you can install the plugin with:
+
+```bash
+sunbeam enable validation
+```
+
+This plugin is also related to `observability` plugin.
+
+## Contents
+
+This plugin will install [tempest-k8s](https://opendev.org/openstack/sunbeam-charms/src/branch/main/charms/tempest-k8s), and provide the following command line interface:
```

Comment:
Though, I would suggest not showing the usage output from `sunbeam validation`, since it's subjected to change easily. I would instead ask the reader to run `sunbeam validation --help` to learn more